app/assets/stylesheets/components/_pagination.scss

```diff
@@ -0,0 +1,14 @@
+.pagination {
+  margin-bottom: 3rem;
+
+  .page-link {
```

Comment:
is `link` an element of the `.page` block, if yes, should we name it to follow the [convention on Compass](https://compass.nimblehq.co/development/code-conventions/css/#class-names)

```css
   .page__link {
   }
   
   .page__item {
    }
```

Reply:
It is not, it is the bootstrap class name.